web/client/epics/map.js

```diff
@@ -93,6 +95,15 @@ const handleCreationLayerError = (action$, store) =>
             ]) : Rx.Observable.empty();
         });
 
+// Epic to determine which trigger to use when map loads
```

Comment:
This epic have to be moved go in mapInfo epics. 
I suggest to intercept **also** on the `setMapTrigger` event, so we have a single entry point  to register or unregister the identify tool, and you can remove the call the registerEventListener from the component.